RegistrationLib/__init__.py

```diff
@@ -3,7 +3,11 @@
 from Landmarks import *
 from RegistrationState import *
 from RegistrationPlugin import *
-from AffinePlugin import *
-from ThinPlatePlugin import *
-from LocalBRAINSFitPlugin import *
-from LocalSimpleITKPlugin import *
+
+for plugin in [
+  'Affine',
+  'ThinPlate',
+  'LocalBRAINSFit',
+  'LocalSimpleITK'
+  ]:
+  registerRegistrationPlugin('RegistrationLib.%sPlugin' % plugin)
```

Comment:
Waiting I answer that question, I just did an other PR #14 including the fix for the failing generic test. Would be great to integrate it.